biosimulations/apps/dispatch-service/src/app/app.controller.ts

```diff
@@ -74,8 +79,8 @@ export class AppController {
     this.hpcService.dispatchJob(
       hpcTempDirPath,
       sbatchPath,
-      omexPath,
-      data.filename
+      // omexPath,
+      data.omexFileName
     );
 
     return { message: 'Simulation dispatch started.' };
```

Comment:
This is the message that is going to be sent back. So the above api should ensure that it gets this message and handles the error if not. 
The message should probably be changed to something more informative like the time or other info about the status

Reply:
Resolving this because the code is updated.